src/java/picard/analysis/CollectWgsMetrics.java

```diff
@@ -99,7 +99,8 @@
     @Option(shortName = "MQ", doc = "Minimum mapping quality for a read to contribute coverage.", overridable = true)
     public int MINIMUM_MAPPING_QUALITY = 20;
 
-    @Option(shortName = "Q", doc = "Minimum base quality for a base to contribute coverage.", overridable = true)
+    @Option(shortName = "Q", doc = "Minimum base quality for a base to contribute coverage. N's in the SEQ will be excluded" +
+            "from coverage and counted as though they are excluded for quality regardless of the value of this parameter.", overridable = true)
```

Comment:
This was hard to read.

'N's will be counted as having a base quality of negative infinity and will be excluded from coverage.`?